flux-image-automation-controller.yaml

```diff
@@ -0,0 +1,54 @@
+package:
+  name: flux-image-automation-controller
+  version: 0.33.1
+  epoch: 0
+  description: GitOps Toolkit controller that patches container image tags in Git
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - llvm-lld
+      - gcc
+      - clang-15
+  environment:
+    CGO_ENABLED: 1
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/fluxcd/image-automation-controller
+      tag: v${{package.version}}
+      expected-commit: aeb528929f06afcd2f48a26cf2521b87f4b7be5b
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/bin
+      CGO_ENABLED=1 CGO_LDFLAGS="-static -fuse-ld=lld" go build \
```

Comment:
Just curious about the reason for the `CGO_ENABLED=1` here, since I see it under the `environment:` section up above, too. Do we have a preference on which place we use it?

Reply:
I was working an another image probably we can remove the environment